web/src/message_list_view.js

```diff
@@ -356,6 +356,9 @@ export class MessageListView {
         const is_hidden = message_container.is_hidden;
         const status_message = Boolean(message_container.status_message);
         const edit_history_details = analyze_edit_history(message_container.msg, last_edit_timestr);
+        message_container.editing_in_left_col = !include_sender && !is_hidden;
+        message_container.editing_alongside_sender = include_sender && !status_message;
+        message_container.editing_status_msg = include_sender && status_message;
```

Comment:
How do these values differ from `edited_in_left_col` and friends?

Reply:
their values are same, but they differ from the `edited_in_left_col` in when their values are set. Like for `edited_in_left_col`, value initialized only after an edit is made, while for `editing_in_left_col`, I made a small modification to set its value every time an individual message is rendered.

Reply:
OK. This is probably at least worth a comment ... but it might be better to just do a preparatory refactor to define properties like `message_edit_notices_in_left_col` for location logic, and then have a single variable for the edited content... Want to try doing that in a little preparatory PR that we can merge early?

Reply:
resolved in latest update!